docs/installation/manual/installing_composer_drush_and_drupal.md

```diff
@@ -0,0 +1,128 @@
+# Installing Composer, Drush, and Drupal
+
+## In this section, we will install:
+
+- [Composer](https://getcomposer.org/) at its current latest version, the package manager that will allow us to install PHP applications
+- The composer installer from [drupal-composer/drupal-project](https://github.com/drupal-composer/drupal-project), which will install, among other things:
+    - [Drush 9](https://www.drush.org/) at its latest version, the command-line PHP application for running tasks in Drupal
+    - [Drupal 8](https://www.drupal.org/) at its latest version, the content management system Islandora uses for content modelling and front-end display
+
+## Composer 1.x
+
+### Download and install Composer
+
+Composer provides PHP code that we can use to install it. After downloading and running the installer, we’re going to move the generated executable to a place in `$PATH`, removing its extension:
+
+```bash
+curl "https://getcomposer.org/installer" > composer-install.php
+chmod +x composer-install.php
+php composer-install.php
+sudo mv composer.phar /usr/local/bin/composer
+```
+
+## Drush 9 and Drupal 8
+
+### Clone `drupal-project` and run `composer install`
+
+Before we can fully install Drupal, we’re going to need to clone `drupal-project` and provision it using Composer. We’re going to install it into the `/opt` directory:
+
+```bash
+# Start by giving Drupal somewhere to live. The Drupal project is installed to
+# an existing, empty folder.
+sudo mkdir /opt/drupal
+sudo chown www-data:www-data /opt/drupal
+sudo chmod 775 /opt/drupal
+# Clone drupal-project and build it in our newly-created folder.
+git clone https://github.com/drupal-composer/drupal-project.git
```

Comment:
Doing this in my home directory caused a number of composer warning about "Cannot create cache directory" when running composer as www-data.